guides/components/component-arguments-and-html-attributes.md

```diff
@@ -1,144 +1,115 @@
-Components become useful building blocks of our app if we make them _reusable_. When we reuse components efficiently, we can avoid having to recreate parts of our app again and again. If you want to _reuse_ a component in multiple places, you'll need a way to template out parts of it.
+Les composants deviennent des éléments utiles d'une application quand on les rend "réutilisables". Lorsqu'on réutilise efficacement les composants, on peut éviter d'avoir à recréer des parties de l'app encore et encore. Si vous voulez "réutiliser" un composant à plusieurs endroits, vous aurez besoin d'un moyen d'en abstraire certaines parties.
```

Comment:
- Nitpicking but, do we want to keep the context of meaning "our app" since the guide goes through building while "holding the hand" of the reader. To make it more personal.

- It might seem insignificant but something like "briques" or any word less general than "élément" would sound more adequate to me. But I won't fight against that.

```suggestion
Les composants deviennent des blocs utiles de notre application quand on les rend "réutilisables". Lorsqu'on réutilise efficacement les composants, on peut éviter d'avoir à recréer des parties de l'app encore et encore. Si vous voulez "réutiliser" un composant à plusieurs endroits, vous aurez besoin d'un moyen d'en abstraire certaines parties.
```

Reply:
I'll go with "briques" to not introduce any confusion with components blocks 👍